code_snippets/ts/src/develop/my_virtual_object.ts

```diff
@@ -6,6 +6,9 @@ const myVirtualObject = restate.object({
         myHandler: async (ctx: restate.ObjectContext, greeting: string) => {
             return `${greeting} ${ctx.key}!`;
         },
+        myConcurrentHandler: async (ctx: restate.ObjectSharedContext, greeting: string) => {
```

Comment:
this is not enough, you need this thing here https://github.com/restatedev/sdk-typescript/blob/main/packages/restate-sdk-examples/src/object.ts#L34